lib/live_view_native_swift_ui/modifiers/text_input_and_output/privacy_sensitive.ex

```diff
@@ -0,0 +1,7 @@
+defmodule LiveViewNativeSwiftUi.Modifiers.PrivacySensitive do
+  use LiveViewNativePlatform.Modifier
+
+  modifier_schema "privacy_sensitive" do
+    field :sensitive, Ecto.Enum, values: ~w(true false)a
```

Comment:
Please use a `:boolean` type.
```suggestion
    field :sensitive, :boolean
```